src/ssd1306-128x64-element.ts

```diff
@@ -0,0 +1,93 @@
+// Reference: https://cdn-learn.adafruit.com/assets/assets/000/036/494/original/lcds___displays_fabprint.png?1476374574
+import { customElement, html, LitElement, property, SVGTemplateResult } from 'lit-element';
+
+@customElement('wokwi-ssd1306-128x64-element')
+export class Ssd1306128x64Element extends LitElement {
+  width = 150;
+  height = 116;
+  @property() imageData: ImageData = new ImageData(128, 64);
+
+  putGivenImageData() {
+    const canvas: HTMLCanvasElement | null | undefined = this.shadowRoot?.querySelector('canvas');
+    const ctx = canvas?.getContext('2d');
+    if (!ctx) return;
+    ctx.putImageData(this.imageData, 0, 0);
+  }
+
+  firstUpdated() {
+    this.putGivenImageData();
+  }
+
+  render(): SVGTemplateResult {
+    const { width, height } = this;
+    return html`<svg width="${width}" height="${height}" xmlns="http://www.w3.org/2000/svg">
+      <g>
+        <rect stroke="#BE9B72" fill="#025CAF" x=".5" y=".5" width="148" height="114" rx="13" />
+
+        <g transform="translate(6 6)" fill="#59340A" stroke="#BE9B72">
+          <circle cx="130" cy="6" r="5.5" />
+          <circle cx="6" cy="6" r="5.5" />
+          <circle cx="130" cy="96" r="5.5" />
+          <circle cx="6" cy="96" r="5.5" />
+        </g>
+
+        <g transform="translate(8 26)">
+          <!-- 128 x 64 screen -->
+          <rect fill="#1A1A1A" width="128" height="64" />
+          <!-- image holder -->
+          <foreignObject width="128" height="64">
+            <canvas width="128" height="64"></canvas>
+          </foreignObject>
+        </g>
+
+        <!-- All texts -->
+        <g
+          fill="#FFF"
+          text-anchor="middle"
+          font-size="5"
+          font-weight="300"
+          font-family="MarkerFelt-Wide, Marker Felt"
```

Comment:
These fonts are not available on all computers - let's add a fallback from one of the web-safe fonts (https://www.w3schools.com/cssref/css_websafe_fonts.asp)

Reply:
done